contrib/docker/Dockerfile.debian

```diff
@@ -0,0 +1,71 @@
+# This Dockerfile creates an debian image with avocado framework installed
+# from source
+#
+# VERSION 0.1
+################
+## Usage example:
+# git clone github.com/avocado-framework/avocado.git avocado.git
+# cd avocado.git
+## Make some changes
+# patch -p1 < MY_PATCH
+## Finally build a docker image
+# docker build --force-rm -t debian-avocado-custom -f contrib/docker/Dockerfile.debian .
+## Run test inside the docker image
+# avocado run --docker fedora-avocado-custom passtest.py
+#
+FROM debian
+MAINTAINER Dmitry Monakhov dmonakhov@openvz.org
+# Install and clean in one step to decrease image size
+
+RUN apt-get update && \
+    echo install avocado def packages && \
+    apt-get install -y --no-install-recommends \
+	    git \
+	    rsync \
+	    make \
+	    gdebi-core \
+	    pkg-config \
+	    libvirt-dev \
+	    python-dev \
+	    python-lzma \
+	    python-pip \
+	    python-pystache \
+	    python-setuptools \
+	    python-stevedore  \
+	    python-yaml && \
+    echo install extra avocado packages && \
+    apt-get install -y --no-install-recommends \
+	    ansible \
+	    emacs-nox \
```

Comment:
Even though I love emacs, it's here primarily for the author's convenience, right?  Again, since it's an example, it's fine.